Authorization/Authorization/Presentation/Login/SignInViewModel.swift

```diff
@@ -29,20 +50,78 @@ public class SignInViewModel: ObservableObject {
             }
         }
     }
+    var oauthswift: OAuth2Swift?
     
     private let interactor: AuthInteractorProtocol
     let router: AuthorizationRouter
+    let config: Config
     let analytics: AuthorizationAnalytics
     private let validator: Validator
+    private var safariDelegate: WebLoginSafariDelegate?
     
     public init(interactor: AuthInteractorProtocol,
                 router: AuthorizationRouter,
                 analytics: AuthorizationAnalytics,
+                config: Config,
                 validator: Validator) {
         self.interactor = interactor
         self.router = router
         self.analytics = analytics
+        self.config = config
         self.validator = validator
+        self.webLoginAttempted = false
```

Comment:
Is it needed to initialize `webLoginAttempted` with `false` on init if the variable is already initialized `var webLoginAttempted: Bool = false`?

Reply:
Nope-- that got left in when I was trying a few things. I'll remove it.